test/indexed-images-spec.js

```diff
@@ -0,0 +1,152 @@
+let fs = require("fs");
```

Comment:
I'm not sure if it would make sense to add another loop in the main file or extract the testing logic so it was clear how this differs